src/schema/feeds.ts

```diff
@@ -857,15 +858,16 @@ const feedResolverV2: IFieldResolver<
       args: FeedArgs & { version: number; feedId?: string },
       page,
     ) =>
-      generatePersonalizedFeed({
-        con: ctx.con,
-        pageSize: page.limit,
-        offset: page.offset,
-        feedVersion: args.version,
-        userId: ctx.userId || ctx.trackingId,
-        feedId: args.feedId,
+      (args.feedId
```

Comment:
Not super important, but can't we clean out some logic of this feedResolver as well?
Seems very heavy still.

Reply:
Will do!